src/Conversion/ONNXToStablehlo/NN/ConvTranspose.cpp

```diff
@@ -88,7 +88,6 @@ struct ONNXConvTransposeOpLoweringToStablehlo : public ConversionPattern {
     assert(isRankedShapedType(inputOperand.getType()) &&
            "Expected Ranked ShapedType");
     ShapedType inputType = inputOperand.getType().cast<ShapedType>();
-    Type elemType = inputType.getElementType();
```

Comment:
FYI- I removed this because we were getting warnings during compile time that this is not used. I checked it is not used after line 91 and already defined above.